src/h5web/vis-packs/core/shared/PanZoomMesh.tsx

```diff
@@ -91,11 +93,16 @@ function PanZoomMesh(): ReactElement {
     moveCameraTo(camera.position.x, camera.position.y);
   }, [camera, moveCameraTo]);
 
+  const wheelCaptureRef = useWheelCapture();
+
   return (
-    <mesh {...{ onPointerMove, onPointerUp, onPointerDown, onWheel }}>
-      <meshBasicMaterial attach="material" opacity={0} transparent />
-      <planeGeometry attach="geometry" args={[width, height]} />
-    </mesh>
+    <>
+      <mesh {...{ onPointerMove, onPointerUp, onPointerDown, onWheel }}>
+        <meshBasicMaterial attach="material" opacity={0} transparent />
+        <planeGeometry attach="geometry" args={[width, height]} />
+      </mesh>
+      <Html ref={wheelCaptureRef} />
```

Comment:
The other option would be to have the `ref` on the `div` that R3F creates and wraps the canvas with, but it doesn't seem to be possible. We could wrap the canvas in an extra div (the current wrapper, `vis`, has padding for the axes...), but it's a bit hacky with regard to flexbox...

This option is simpler, and it makes more sense, since it's here that the zoom is implemented in the first place.